node/scripts/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/bash -x
```

Comment:
why is there a space between the hashbang and the `/bin/bash`?, and why the `-x`?

Reply:
v

Reply:
v